Extension/src/LanguageServer/referencesTreeDataProvider.ts

```diff
@@ -121,7 +121,7 @@ export class ReferencesTreeDataProvider implements vscode.TreeDataProvider<TreeN
                 if (!element.referenceText) {
                     throw new Error("Null or undefined referenceText in getTreeItem()");
                 }
-                if (!element.referenceType) {
+                if (element.referenceType === undefined) {
                     throw new Error("Null or undefined referenceType in getTreeItem()");
```

Comment:
Can you assign null to a `?` property?  I think it can only be `undefined`

Reply:
Maybe we enable https://palantir.github.io/tslint/rules/no-null-keyword/ later on.

Reply:
I corrected those error messages.  Hypothetically, it's possible for external code (i.e. VS Code APIs) to return null or undefined from APIs that do not indicate they might.  (i.e. for templates  like get<type> in settings, if we support a null setting and don't use get<type | null> when we call it, we may still get null back).